src/Microsoft.AspNetCore.Sockets.Client/Message.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO.Pipelines;
+
+namespace Microsoft.AspNetCore.Sockets.Client
+{
+    public struct Message : IDisposable
+    {
+        public Format MessageFormat { get; }
```

Comment:
If we want the connection to be IChannel<Message> then we should add make using this struct more convenient (e.g. add Message(string data)/Message(byte[] data) ctors etc)